packages/docusaurus-theme-classic/src/theme/DocItem/index.tsx

```diff
@@ -112,6 +112,7 @@ function DocItem(props: Props): JSX.Element {
                     <div className="col text--right">
                       <em>
                         <small>
+                          {/* TODO: wait for using interpolation in translation function */}
```

Comment:
Yes, this is a missing piece that I'd like to add, not sure how yet.

React-intl has this feature to handle these kind of cases, and allow to even inject components into a translated string (like a link). But I want to add a very minimal runtime, not something too heavy with many features we don't need. If you know a good lib for that let me know

Reply:
not 100% related but kind of the same problem if you want to interleave links in a translated string.

On Jest website I used a markdown parser on the client (v1 had a MarkdownBlock component), and it allows to wire the Docusaurus link for the markdown links somehow.

![image](https://user-images.githubusercontent.com/749374/107773613-346eab00-6d3e-11eb-84ef-1d228266c6d9.png)

![image](https://user-images.githubusercontent.com/749374/107773707-56682d80-6d3e-11eb-9c59-983647af3f32.png)

![image](https://user-images.githubusercontent.com/749374/107773654-43555d80-6d3e-11eb-9486-7750bc4198fe.png)

But if we make all this possible through the Translate component, I'd like to avoid the cost of shipping a full md parser to the client.